CMakeLists.txt

```diff
@@ -81,10 +81,10 @@ target_link_libraries(GutterTree PUBLIC gtest GraphZeppelinCommon)
 # different OSes have different ways of efficiently pre-allocating large files
 if (UNIX AND NOT APPLE)
   message(STATUS "Enabling Fallocate for a linux system")
-  target_compile_options(GutterTree PRIVATE -DLINUX_FALLOCATE)
+  target_compile_options(GutterTree PRIVATE -fopenmp -DLINUX_FALLOCATE)
```

Comment:
Make these flags PUBLIC rather than PRIVATE so that associated executables will automatically use them